cfme/common/vm.py

```diff
@@ -467,6 +467,8 @@ def create(self, vm_name, provider, form_values=None, cancel=False,
             check_existing: verify if such vm_name exists
             find_in_cfme: verify that vm was created and appeared in CFME
             wait: wait for vm provision request end
+            extra_msg: extra message appended to the description of the request.
+            approve: If a request for the vm/instance need's approval.
```

Comment:
**REQUIRED:** I think this should be altered slightly. I'm not even sure I wanted this in here as an option, though I see that we need to do it before the wait. Perhaps we name it ```auto_approve```? And put the comment as something like ```if true the request is approved before waiting for completion```